upload/src/addons/SV/WarningImprovements/Entity/WarningDefault.php

```diff
@@ -50,10 +50,10 @@ public static function getStructure(Structure $structure)
             'warning_default_id' => ['type' => self::UINT, 'autoIncrement' => true, 'nullable' => true],
             'threshold_points'   => ['type' => self::UINT, 'required' => true, 'max' => 65535],
             'expiry_type'        => [
-                'type'          => self::STR, 'default' => 'never',
-                'allowedValues' => ['never', 'days', 'weeks', 'months', 'years']
```

Comment:
Please allow "never", it then needs to set the suggested warning expiry to 0 (aka never)

Reply:
Looks like the XF1's UI wasn't completed or reverted at some stage :(

Reply:
I checked on XF1 board to see if there was any 'never' value but nope so I just removed it from here as well guessing it a bug